kitsune/questions/api.py

```diff
@@ -389,18 +387,6 @@ def auto_tag(self, request, pk=None):
         question.auto_tag()
         return Response(status=status.HTTP_204_NO_CONTENT)
 
-    @action(detail=True, methods=["post"])
-    def attach_images(self, request, pk=None):
```

Comment:
why are dropping this completely? Not used anywhere?

Reply:
Yep, and somewhat confusing as it dupes the code that happens in the question save method